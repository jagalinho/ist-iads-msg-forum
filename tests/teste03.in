A 123 Mensagem de teste
A 123 Outra mensagem de teste
A 567 Ainda outra mensagem de teste
A 890 A roda do autocarro roda roda
T
A 890 A roda do autocarro roda roda roda
T
S
A 890 A roda do autocarro roda
A 123 A roda do autocarro roda
S
X