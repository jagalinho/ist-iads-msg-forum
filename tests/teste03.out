*MOST ACTIVE USER:123:2
*MOST ACTIVE USER:123:2
*MOST ACTIVE USER:890:2
*SORTED MESSAGES:5
890:A roda do autocarro roda roda
890:A roda do autocarro roda roda roda
567:Ainda outra mensagem de teste
123:Mensagem de teste
123:Outra mensagem de teste
*SORTED MESSAGES:7
123:A roda do autocarro roda
890:A roda do autocarro roda
890:A roda do autocarro roda roda
890:A roda do autocarro roda roda roda
567:Ainda outra mensagem de teste
123:Mensagem de teste
123:Outra mensagem de teste
7
